modules/aruco/tutorials/aruco_detection/aruco_detection.markdown

```diff
@@ -158,45 +163,43 @@ An example of marker detection:
+- The first parameter is the image containing the markers to be detected.
+- The second parameter is the dictionary object, in this case one of the predefined dictionaries (`DICT_6X6_250`).
+- The detected markers are stored in the `markerCorners` and `markerIds` structures:
+    - `markerCorners` is the list of corners of the detected markers. For each marker, its four
     corners are returned in their original order (which is clockwise starting with top left). So, the first corner is the top left corner, followed by the top right, bottom right and bottom left.
-    - ```markerIds``` is the list of ids of each of the detected markers in ```markerCorners```.
-    Note that the returned ```markerCorners``` and ```markerIds``` vectors have the same sizes.
-- The fourth parameter is the object of type ```DetectionParameters```. This object includes all the
-parameters that can be customized during the detection process. This parameters are commented in
-detail in the next section.
-- The final parameter, ```rejectedCandidates```, is a returned list of marker candidates, i.e. those
-squares that have been found but they do not present a valid codification. Each candidate is also
-defined by its four corners, and its format is the same than the ```markerCorners``` parameter. This
-parameter can be omitted and is only useful for debugging purposes and for 'refind' strategies (see ```refineDetectedMarkers()``` ).
-
-
-The next thing you probably want to do after ```detectMarkers()``` is checking that your markers have
+    - `markerIds` is the list of ids of each of the detected markers in `markerCorners`.
+    Note that the returned `markerCorners` and `markerIds` vectors have the same size.
+- The fourth parameter is the object of type `DetectionParameters`. This object includes all the
+parameters that can be customized during the detection process. These parameters are explained in the next section.
+- The final parameter, `rejectedCandidates`, is a returned list of marker candidates, i.e.
+shapes that were found and considered but did not contain a valid marker. Each candidate is also
+defined by its four corners, and its format is the same as the `markerCorners` parameter. This
+parameter can be omitted and is only useful for debugging purposes and for ‘refind’ strategies (see `refineDetectedMarkers()` ).
+
+
+The next thing you probably want to do after `detectMarkers()` is check that your markers have
 been correctly detected. Fortunately, the aruco module provides a function to draw the detected
-markers in the input image, this function is ```drawDetectedMarkers()```. For example:
+markers in the input image, this function is `drawDetectedMarkers()`. For example:
 
 @code{.cpp}
-    cv::Mat outputImage
-    cv::aruco::drawDetectedMarkers(image, markerCorners, markerIds);
+	cv::Mat outputImage = inputImage.clone();
+	cv::aruco::drawDetectedMarkers(outputImage, markerCorners, markerIds);
```

Comment:
Please avoid using tabs (in code and tutorials).
Zero indentation should fit well here.

Reply:
Removed all tabs.